osu.Game/Overlays/BeatmapSet/BeatmapPicker.cs

```diff
@@ -274,8 +274,9 @@ public DifficultySelectorButton(APIBeatmap beatmapInfo)
                             Alpha = 0.5f
                         }
                     },
-                    icon = new DifficultyIcon(beatmapInfo, shouldShowTooltip: false)
+                    icon = new DifficultyIcon(beatmapInfo.Ruleset)
                     {
+                        Current = { Value = new StarDifficulty(beatmapInfo.StarRating, 0) },
```

Comment:
Is it intended that this doesn't just use `DifficultyIcon(BeatmapInfo)` and disable tooltip explicitly?

Reply:
This was before I added the `ShowTooltip`. I'll update accordingly.